variables.tf

```diff
@@ -401,6 +401,16 @@ variable "cni_network_provider" {
     default = "OpenshiftSDN"
 }
 
+variable "clusterNetwork_CIDR" {
+    description = "blocks of IP addresses from which pod IP addresses are allocated."
+    default = "10.128.0.0/14"
+}
+
+variable "serviceNetwork" {
+    description = "blocks of IP addresses from which pod IP addresses are allocated."
```

Comment:
Change in description needed? (to distinguish from `clusterNetwork_CIDR`)

Reply:
done